### PR TITLE
[zelos] Multi-line rendering

### PR DESCRIPTION
**Pull request by samelhusseini**
## The basics
- [x] I branched from develop
- [x] My pull request is against develop
- [x] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)

### Resolves

Fixes https://github.com/google/blockly/issues/3494
Fixes https://github.com/google/blockly/issues/2978

### Proposed Changes

Add a new ``isMultiLine`` property on the info object, and based on that decide whether or not to create a new row.

Size the negative spacing for blocks with output connections based on the radius created by the height of the block in order to maintain a ``SMALL_PADDING`` padding around.

We could use the isInline property, we would just need override code to instead render inline inputs instead of external inputs all over, both in the info object and drawer. I thought it would be cleaner to create a new property.

### Reason for Changes

Zelos rendering

### Test Coverage

Tested geras and zelos in playground.
Tested test blocks in playground.

Tested on:

### Additional Information

Screen Shot 2019-12-10 at 10 30 04 AM

Screen Shot 2019-12-10 at 10 34 01 AM

### CODE REVIEW

File: core/renderers/common/info.js [hunk 1]
```diff
@@ -138,6 +138,13 @@ Blockly.blockRendering.RenderInfo = function(renderer, block) {
    */
   this.rows = [];
 
+  /**
+   * The number of active rows on a block.
```
Comment (rachel-fenichel) [inline core/renderers/common/info.js]: Update the comment to say that while creating rows, it' the currently active one, and after creating rows it's the total number of rows on the block.

Comment (samelhusseini) [reply core/renderers/common/info.js]: I think instead I'm going to call it inputRowCount as I'm only interested in the number of input rows.

Comment (samelhusseini) [reply core/renderers/common/info.js]: I'm not sure about that comment. It's not the currently active one, it's not even the index of the row of the active row. It's the current count of input rows created.

Comment (samelhusseini) [reply core/renderers/common/info.js]: How does ``The total number of input rows added onto the block.`` sound?

Comment (rachel-fenichel) [reply core/renderers/common/info.js]: Sounds good. I thought you were using the value to determine the negative padding for each row as you went, but you're only using whether it's greater than 1.

**Pull request merged**